mne/viz/_3d.py

```diff
@@ -388,29 +409,46 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
         mri_trans = invert_transform(head_mri_t)
 
     # both the head and helmet will be in MRI coordinates after this
-    meg_picks = pick_types(info, meg=True, ref_meg=ref_meg)
     surfs = dict()
-    if show_head:
+    if head:
         subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
         surfs['head'] = get_head_surf(subject, source=source,
                                       subjects_dir=subjects_dir)
     if 'helmet' in meg_sensors and len(meg_picks) > 0 and \
             (ch_type is None or ch_type == 'meg'):
         surfs['helmet'] = get_meg_helmet_surf(info, head_mri_t)
+    if brain is None:
+        if len(ecog_picks) > 0 and subject is not None:
```

Comment:
not strictly necessary but I find it reads more easily (explicit instead of implicit)